docs/Chipyard-Basics/Debugging-RTL.rst

```diff
@@ -0,0 +1,85 @@
+Debugging RTL
+======================
+
+While the packaged Chipyard configs and RTL have been tested to work,
+users will typically want to build custom chips by adding their own
+IP, or by modifying existing Chisel generators. Such changes might introduce
+bugs. This section aims to run through a typical debugging flow
+using Chipyard. We assume the user has a custom SoC configuration,
+and is trying to verify functionality by running some software test.
+
```

Comment:
I think there should be an extra section here, talking about the fact you should start with making sure your software is correct by running it on a functional simulator (spike for example)

Reply:
Done